### ISSUE TITLE
[Feature] Move logout button to the end of list

### ISSUE THREAD

**Issue by ketozhang** [FreshRSS/FreshRSS#5716]
**Is your feature request related to a problem? Please describe.**
I’m always frustrated<sup>1</sup> when I click the gear icon and accidentally log out. I also accidentally clicked logout from the settings page.

<sup>[1]: I love how the issue template started my sentence </sup>

**Describe the solution you’d like**
Please move logout to the last item in settings.  It is common practice, bet its part of some UX consensus---for example, github:
![image](https://github.com/FreshRSS/FreshRSS/assets/13506487/d30808ad-a6b8-440e-8c56-ba390d067f43)

**Comment by Frenzie** [comment]
I'm inclined to agree. 👍

**Comment by vdbhb59** [comment]
Agreed. I too get logged out due to page getting aligned upon full load. Very frustrating.

**Comment by math-GH** [comment]
Thanks for giving feedback so that we could improve the UX.

Before we discuss a solution let me understand the issue better.

Which theme do you use?
Which FreshRSS version?
Mobile or Desktop view?
What exactly do you do? Could you please describe step by step your actions or could you please record it as video/GIF?

**Comment by vdbhb59** [comment]
> Thanks for giving feedback so that we could improve the UX.
> 
> Before we discuss a solution let me understand the issue better.
> 
> Which theme do you use? Which FreshRSS version? Mobile or Desktop view? What exactly do you do? Could you please describe step by step your actions or could you please record it as video/GIF?

Responses below:
Irrespective of the theme the logout button is at 2nd position
1.21.0
Mobile/Desktop - both
For me happens mostly when loading a page and clicking on it in error upon trying to click on first instance
It also happens when in error or by mistake the logout button gets clicked while trying to click some other button

**Comment by ketozhang** [comment]
Ditto to @vdbhb59 with the difference that I am use https://feeds.flossboxin.org.in (I am a fresh user just trying this out before self-hosting).

**Comment by vdbhb59** [comment]
> Ditto to @vdbhb59 with the difference that I am use https://feeds.flossboxin.org.in (I am a fresh user just trying this out before self-hosting).

That is my instance only (I run it). So, similar issue. Though, I found it same on my personal host too on a different server.

**Comment by Frenzie** [comment]
I've never accidentally logged out but I can see how it's dangerously close to other things. But also, the position of close/logout everywhere else conditions us to look at the bottom.

**Comment by HarikaB11** [comment]
Hi @ketozhang , I would like to work on this issue.

**Comment by Alkarex** [comment]
Go ahead @ketozhang 👍🏻

**Comment by MananDhiman** [comment]
Please take a look at my commit [here](https://github.com/MananDhiman/FreshRSS/commit/2cbfc5d5e9f27462f2087aa40fe4b0a5e0fee411) and if any changes let me know

If everything is okay I'll create a pull request

It's my first pull request so I appreciate all help

**Comment by Alkarex** [comment]
@MananDhiman Just make a PR. It makes it easier to provide feedback, further edits, and tests

**Comment by aledeg** [comment]
Should we close this considering #5907, #5966, and #6047?

**Comment by Alkarex** [comment]
> Should we close this considering [#5907](https://github.com/FreshRSS/FreshRSS/pull/5907), [#5966](https://github.com/FreshRSS/FreshRSS/discussions/5966), and [#6047](https://github.com/FreshRSS/FreshRSS/pull/6047)?

I do not have any strong feeling about this one. Maybe @math-GH ?

**Comment by Frenzie** [comment]
It does seems so. ;-)
https://github.com/FreshRSS/FreshRSS/pull/5907#issuecomment-1826835083

**Comment by ketozhang** [comment]
Curious why close an issue when all linked PRs were rejected or in draft (not merged).

Not planned is a better status

**Comment by Alkarex** [comment]
Fixed :-P
